ethereum-consensus/src/capella/block_processing.rs

```diff
@@ -103,7 +130,7 @@ pub fn process_operations<
                 expected: expected_deposit_count,
                 count: body.deposits.len(),
             },
-        )))
+        )));
```

Comment:
are you running the nightly `cargo`?

if you have `just` installed, you can run `just fmt` and it will execute the right command for you

Reply:
I put changes by running the lint from the ci. Will add just